### ISSUE TITLE
Panel on right side not showing when opening new battles.

### ISSUE THREAD

**Issue by Pospesel** [Zaraka/GruntArbiter#2]
https://i.imgur.com/cjlTs67.jpg

https://i.imgur.com/dLUn5dX.jpg

**Comment by Zaraka** [comment]
I'll go with.. "that's a feature".
The intended functionality is. Right panel can't be showed when there isn't selected any item in any tables (characters, vehicles, structures e.t.c.).
As soon as there is selected item, right panel will show with details of that item. Without selection there isn't anything to show in side panel.

But I used to have problems where even after selecting items in tables, right panel didn't show up. Is this the problem?

**Comment by Pospesel** [comment]
Yes, no matter who or what I select, the panel never shows up. I actually abandoned my Windows installation and booted up Ubuntu, but I'm still having the same issue there.

**Comment by Zaraka** [comment]
Can you replicate this issue everytime? If so, please upload your campaign file, ill look into it

**Comment by Pospesel** [comment]
[Electric Sheep.zip](https://github.com/Zaraka/GruntArbiter/files/1935172/Electric.Sheep.zip)

**Comment by Zaraka** [comment]
I looked into it, but I can't replicate it. Master branch is kinda a few commits ahead so maybe it got fixed. I'll leave this open and notify you when I release new version to ty and see if it helped.

You can always try to create new campaign file, even though it's pain in the ass, this app really needs import functions.
